HIRONX python process unusable when ros_bridge is not available

```
$ rosversion hironx_ros_bridge
1.1.5
$ ipython -i `rospack find hironx_ros_bridge`/scripts/hironx.py -- --host hiro
:
[hrpsys.py]   setupLogger : servoState arleady exists in DataLogger
[rtm.py]    Connect RobotHardware0.servoState - log.RobotHardware0_servoState
[hrpsys.py] setup joint groups
[hrpsys.py] initialized successfully
[ERROR] [1454469015.010880313]: Robot semantic description not found. Did you forget to define or remap '/robot_description_semantic'?
[ERROR] [1454654447.415352066]: [registerPublisher] Failed to contact master at [localhost:11311].  Retrying...
```

And once ros_bridge becomes available, the python process crashes.

```
[ INFO] [1454469006.680039378]: Connected to master at [localhost:11311]
Segmentation fault
```

--- a/hironx_ros_bridge/src/hironx_ros_bridge/ros_client.py
+++ b/hironx_ros_bridge/src/hironx_ros_bridge/ros_client.py
@@ -71,14 +71,14 @@ class ROS_Client(RobotCommander):
         '''
         @param jointgroups [str]: Deprecated. No need after version 1.1.4 onward.
         '''
-        super(ROS_Client, self).__init__()  # This solves https://github.com/start-jsk/rtmros_hironx/issues/300
-
         # if we do not have ros running, return 
         try:
             rospy.get_master().getSystemState()
         except Exception:
             print('[ros_client] ros master is not running, so do not create ros client...')
             return
+
+        super(ROS_Client, self).__init__()  # This solves https://github.com/start-jsk/rtmros_hironx/issues/300
 
         rospy.init_node('hironx_ros_client')
 

--- a/hironx_ros_bridge/src/hironx_ros_bridge/ros_client.py
+++ b/hironx_ros_bridge/src/hironx_ros_bridge/ros_client.py
@@ -75,7 +75,8 @@ class ROS_Client(RobotCommander):
         try:
             rospy.get_master().getSystemState()
         except Exception:
-            print('[ros_client] ros master is not running, so do not create ros client...')
+            from termcolor import colored
+            print(colored('[ERROR][ros_client] ros master is not running, so do not create ros client...', 'red'))
             return
 
         super(ROS_Client, self).__init__()  # This solves https://github.com/start-jsk/rtmros_hironx/issues/300

